contentcuration/contentcuration/viewsets/base.py

```diff
@@ -75,9 +74,11 @@ def id_value_lookup(self, data):
         else:
             # Could alternatively have coerced the list of values to a string
             # but this seemed more explicit in terms of the intended format.
-            return "[{}]".format(
-                ",".join((str(self.get_value(data, attr)) for attr in id_attr))
-            )
+            id_values = (self.get_value(data, attr) for attr in id_attr)
+
+            # For the combined index, use any related objects' primary key
+            combined_index = (idx.pk if hasattr(idx, 'pk') else idx for idx in id_values)
+            return tuple(combined_index)
```

Comment:
I decided to switch to using a `tuple` in order to make doing stuff like [this](https://github.com/learningequality/studio/pull/2682/files#diff-236743df29ba2f73eea30313543672b6884364acbfcacac1df4589a4a423fd66R724) simpler and less brittle.